### PR TITLE
[VBLOCKS-1583] feat: server linting

### PR DESCRIPTION
**Pull request by kpchoy**
## Submission Checklist

 - [x] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code
 - [ ] New unit tests and integration tests have beed added
 - [x] Code coverage is more or equal to the previous coverage
 - [x] A visual inspection of the `Files changed` tab was made prior to submitting the pull request ensuring the style guide was followed
 - [ ] [CI pipeline](https://app.circleci.com/pipelines/github/twilio/twilio-voice-react-native-app) is green
 - [ ] Included screenshot if necessary

## Description

[VBLOCKS-1583](https://issues.corp.twilio.com/browse/VBLOCKS-1583)

## Breakdown

- Add linting to server
- Add server linting step to CircleCI

## Validation

- CI pipeline

## Additional Notes

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -25,6 +28,7 @@
   "dependencies": {
     "axios": "^1.3.6",
     "dotenv": "^16.0.3",
+    "eslint": "^8.43.0",
```
Comment (mhuynh5757) [inline server/package.json:31]: I think this should be a `devDependency`.

**Review by mhuynh5757 — CHANGES_REQUESTED** [review]

File: server/.eslintrc.js
[the remark this answers is not in the document]
Comment (kpchoy) [reply server/.eslintrc.js:1]: I did `yarn eslint --init`. https://eslint.org/docs/latest/use/command-line-interface#--init . Takes you through an "installation wizard" to generate default values.

If you have something else in mind I can change, I just didn't know what defaults to use.

[the remark this answers is not in the document]
Comment (mhuynh5757) [reply server/.eslintrc.js:1]: Nah, that sounds good. I was just curious 😄

File: server/scripts/updateTwimlApp.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (mhuynh5757) [inline server/scripts/updateTwimlApp.js:1]: 👍

**Review by mhuynh5757 — APPROVED** [review]

**Pull request merged**